karma.conf.js

```diff
@@ -1,9 +1,18 @@
 process.env.CHROME_BIN = require('puppeteer').executablePath();
 
+function getSpecs(specList) {
+    if (specList) {
+        return specList.split(',');
+    }
+    return ['test/*.spec.ts'];
+}
+
 module.exports = function (config) {
     config.set({
         frameworks: ['mocha', 'karma-typescript'],
-        files: ['test/**/*.ts', 'src/**/*.ts'],
+        files: ['test/lib/*.ts', 'test/scenes/*.ts', 'src/**/*.ts'].concat(
```

Comment:
I believe I need to add something here to allow for images to load in tests
https://stackoverflow.com/a/21262098/9292680
I tried this earlier and it didn't work